<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewTask"/>
   <addaction name="actionEditTask"/>
   <addaction name="actionTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewTask">
   <property name="icon">
    <iconset>
     <normaloff>resource/img/add.png</normaloff>resource/img/add.png</iconset>
   </property>
   <property name="text">
    <string>NewTask</string>
   </property>
   <property name="toolTip">
    <string>Add new transfer task</string>
   </property>
  </action>
  <action name="actionEditTask">
   <property name="icon">
    <iconset>
     <normaloff>resource/img/list.png</normaloff>resource/img/list.png</iconset>
   </property>
   <property name="text">
    <string>EditTask</string>
   </property>
   <property name="toolTip">
    <string>Edit a task's property</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
